src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/PowCoinViewRuleTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Features.Consensus.CoinViews;
+using Stratis.Bitcoin.Features.Consensus.Rules;
+using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests.Rules.CommonRules
+{
+    /// <summary>
+    /// These tests only cover the first part of BIP68 and not the MaxSigOps, coinview update or scripts verify or calculate block rewards
+    /// </summary>
+    public class PowCoinViewRuleTests 
+    {
+        private Exception caughtExecption;
+        private Mock<ILogger> logger;
+        private const int HeightOfBlockchain = 1;
+        private RuleContext ruleContext;
+        private UnspentOutputSet coinView;
+        private Transaction transactionWithCoinbaseFromPreviousBlock;
+        private readonly PowCoinViewRule rule;
+
+        public PowCoinViewRuleTests()
+        {
+            this.rule = new PowCoinViewRule();
+        }
+
+        [Fact]
+        public void RunAsync_ValidatingATransactionThatIsNotCoinBaseButStillHasUnspentOutputsWithoutInput_ThrowsBadTransactionMissingInput()
+        {
+            this.GivenACoinbaseTransactionFromAPreviousBlock();
```

Comment:
>ATransactionThatIsNotCoinBaseButStillHasUnspentOutputs

but a tx doesn't have any outputs: `new Transaction { Inputs = { new TxIn() } }`

Reply:
My understanding of the first BIP68 check was that it is a transaction, that looks like a coinbase but it is not the first transaction and therefore not the coinbase so it gets rejected.

More than happy to take your advice on improving this to be more realistic